src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -225,7 +225,10 @@ public CheckResult Check(ParseResult parse, RecordType parameterType, ParserOpti
 
             var result = new CheckResult(parse, Config.CultureInfo, binding)
             {
-                Symbols = combinedSymbols,
+                Source = this,
+                Symbols = combinedSymbols, // all symbols
+                Parameters = symbolTable, // just the parameters (provided at runtime)
+                CultureInfo = Config.CultureInfo
```

Comment:
>CultureInfo = Config.CultureInfo [](http://example.com/codeflow?start=16&length=32)

This info has already been passed during initialization.

Reply:
thanks, merge conflict. I can fix